src/routes/amt/userConsent/cancelUserConsentCode.ts

```diff
@@ -0,0 +1,28 @@
+/*********************************************************************
```

Comment:
in terms of naming of these files, i think we can simplify a bit to just cancel.ts since its under a userconsent folder. Same for the others